.changeset/early-peas-listen.md

```diff
@@ -0,0 +1,6 @@
+---
+"@sveltejs/adapter-node": minor
+---
+
+feat: add graceful shutdown
+feat: add systemd socket activation
```

Comment:
We've been making two changes rather than adding two lines to the same changeset in these cases

Reply:
You mean I should mention both changes in one line? I just changed it.

Reply:
Oh, sorry, my first message wasn't very clear. I actually mean two changeset files